espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/StartActivity.kt

```diff
@@ -25,7 +25,7 @@ class StartActivity : RequestHandler<StartActivityParams, Void?> {
 
     @Throws(AppiumException::class)
     override fun handleInternal(params: StartActivityParams): Void? {
-        startActivity(params.appPackage, params.appActivity, params.optionalIntentArguments)
+        startActivity(params.appPackage, params.appActivity, params.optionalIntentArguments, params.launchDisplayId)
```

Comment:
please change startActivity signature, so it accepts a single data object instead of multiple args

Reply:
Something like startActivity(parms)? and add  startActivity in ActivityHelpers.kt to invoke origin startActivty?

Reply:
yes, something like that, the method should accept one param instead of four